systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,63 @@
+{% extends "meetup/base.html" %}
+{% load staticfiles %}
+
+{% block meetup_location_page_content %}
+<div class="row mt40">
+  <h2 class="ml40">Organizers</h2>
+  <div class="col-sm-3">
+  {% for organizer in object.organizers.all %}
+    {% if organizer.profile_picture.name and organizer.profile_picture.name != "False" %}
+      <a href="{{ organizer.get_absolute_url }}">
+       <img src="{{ organizer.profile_picture.url }}" class="profile-pic img-square img-responsive crop_img" alt=""/>
+      </a>
+    {% else %}
+      <a href="{{ organizer.get_absolute_url }}">      
+        <img src="{%static 'img/default.png' %}" class="profile-pic img-square img-responsive crop_img"
+        alt="{{ organizer }} profile picture"/>
+      </a>
+    {% endif %}
+    <a href="{{ organizer.get_absolute_url }}">
+      {{ organizer.user.first_name }} {{ organizer.user.last_name }}
+    </a>
+    </div>
+    {% if forloop.counter|divisibleby:3 %}
```

Comment:
Since the pagination increase, it also make sense to increase the column number a bit (maybe 5 or 6, whatever feels better to you).